### DISCUSSION TITLE
Unable to Connect via Hostname on Local Network

### DISCUSSION THREAD

**Discussion by KernelGhost** [rustdesk/rustdesk#9398] [Q&A]
Hello,

I am currently using Rustdesk to connect to devices on my local network. While connecting directly using IPv4 addresses (e.g., `192.168.0.18`) works without issues, I am unable to establish a connection using hostnames (e.g., `MyPC.local`).

<img width="322" alt="image" src="https://github.com/user-attachments/assets/47df1212-d49d-43c0-98fa-76b565ecfc87">

Interestingly, the same hostname works with other tools like SSH:

```bash
ssh myuser@MyPC.local
```

__Target (Remote Machine)__
- Rustdesk Version: 1.3.0 (2024-08-19 11:23)
- OS: Fedora 40 (GNOME)

**Controller (Local Machine)**
- Rustdesk Version: 1.3.0 (46)
- OS: macOS 14.6.1 (23G93)

__Question__
Is there a way to configure Rustdesk to recognise hostnames (specifically `.local` hostnames) instead of IP addresses alone? If this feature is not supported, are there any known workarounds to enable hostname resolution?

Any help or guidance would be greatly appreciated.

Thank you!

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/5833
